ts/RootContainer.tsx

```diff
@@ -101,7 +105,12 @@ class RootContainer extends React.PureComponent<Props> {
 
         <IONavigationContainer />
 
-        {this.props.isDebugModeEnabled && <VersionInfoOverlay />}
+        {this.props.isDebugModeEnabled && <DebugInfoOverlay />}
+        {/* Just show the pagoPA test indicator if
+        debug mode is disabled */}
```

Comment:
Maybe we should be more expressive here, telling future us that if the debug mode is enable, the same pagoPA overlay is shown inside the `DebugInfoOverlay` otherwise is shown directly here.